### PR TITLE
SOPS: How to setup in-cluster secrets decryption with HashiCorp Vault

### PR DESCRIPTION
**Pull request by souleb**
This adds a new section describing how to setup encryption with HashiCorp Vault.

### CODE REVIEW

File: content/en/docs/guides/mozilla-sops.md [hunk 1]
```diff
@@ -188,6 +188,62 @@ Note that you shouldn't apply the encrypted secrets onto the cluster with kubect
 
 Encrypting with age follows the same workflow than PGP. See the [Kustomization-controller API doc](/docs/components/kustomize/kustomization/#age) for more details.
 
+
+## Encrypting secrets using HashiCorp Vault
+
+Sops needs a valid Vault token in an environment variable named `VAULT_TOKEN` to decrypt the secrets.
```
Comment (stefanprodan) [inline content/en/docs/guides/mozilla-sops.md:194]: We should make kustomize-controller load the token from the secret and feed it to our [SOPS decryptor](https://github.com/fluxcd/kustomize-controller/tree/main/internal/sops).

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @souleb 🏅

File: content/en/docs/guides/mozilla-sops.md (hunk 1 above)
Comment (souleb) [reply content/en/docs/guides/mozilla-sops.md:194]: created an issue: https://github.com/fluxcd/kustomize-controller/issues/516

Comment (stefanprodan) [reply content/en/docs/guides/mozilla-sops.md:194]: thanks

**Pull request merged**